docs/content/connectors/oceanbase-cdc(ZH).md

```diff
@@ -0,0 +1,238 @@
+2. 在 sys 租户中，为 oblogproxy 创建用户。更多信息，参考 [用户管理文档](https://open.oceanbase.com/docs/community/oceanbase-database/V3.1.1/create-user-3)。
+
+    ```bash
+    mysql -h${host} -P${port} -uroot
+    mysql> SHOW TENANT;
+    mysql> CREATE USER ${sys_username} IDENTIFIED BY '${sys_password}';
+    mysql> GRANT ALL PRIVILEGES ON *.* TO ${sys_username} WITH GRANT OPTION;
+    ```
+
+3. 为待监控有租户创建一个租户。此租户用来读取快照数据和变化事件数据。
+4. 使用以下命令，获取 `rootservice_list` 的值。
+
+    ```bash
+    mysql> SHOW PARAMETERS LIKE 'rootservice_list';
+    ```
+
+5. 按照 [oblogproxy 文档](todo) 配置 oblogproxy。
+
+## 创建 OceanBase CDC 表
+
+使用以下命令，创建 OceanBase CDC 表：
+
+```sql
+-- 每 3000 毫秒检查一次                   
+Flink SQL> SET 'execution.checkpointing.interval' = '3s';
+
+-- 在 Flink SQL 中创建 OceanBase 表 `orders`
+Flink SQL> CREATE TABLE orders (
+    order_id     INT,
+    order_date   TIMESTAMP(0),
+    customer_name STRING,
+    price        DECIMAL(10, 5),
+    product_id   INT,
+    order_status BOOLEAN,
+    PRIMARY KEY (order_id) NOT ENFORCED
+) WITH (
+    'connector' = 'oceanbase-cdc',
+    'scan.startup.mode' = 'initial',
+    'username' = 'user@test_tenant',
+    'password' = 'pswd',
+    'tenant-name' = 'test_tenant',
+    'database-name' = 'test_db',
+    'table-name' = 'orders',
+    'hostname' = '127.0.0.1',
+    'port' = '2881',
+    'rootserver-list' = '127.0.0.1:2882:2881',
+    'logproxy.host' = '127.0.0.1',
+    'logproxy.port' = '2983');
+
+-- 从表 orders 中读取快照数据和 binlogs
+Flink SQL> SELECT * FROM orders;
+```
+
+您也可以访问 Flink 官网文档，快速体验将数据从 OceanBase 导入到 Elasticsearch。更多信息，参考 [Flink 官网文档](https://ververica.github.io/flink-cdc-connectors/release-2.2/content/%E5%BF%AB%E9%80%9F%E4%B8%8A%E6%89%8B/oceanbase-tutorial-zh.html)。
+
+## Flink OceanBase CDC 连接器选项
+
+选项 | 是否必选 | 默认值 | 类型 | 描述
+---- | ----- | ------ | ----- | ----
+connector | 是 | 无 | String | 指定要使用的连接器。此处为 `mysql-cdc`。
+scan.startup.mode | 是 | 无 | String | 指定 Flink OceanBase CDC 消费者的启动模式。可取值为 `initial`、`latest-offset` 或`timestamp`。
+scan.startup.timestamp | 否 | 无 | Long | 起始点的时间戳，单位为秒。仅在 `scan.startup.mode` 的值为 `timestamp` 时适用。
+username | 是 | 无 | String | 连接 OceanBase 数据库的用户的名称。
+password | 是 | 无 | String | 连接 OceanBase 数据库时使用的密码。
+tenant-name | 是 | 无 | String | 待监控 OceanBase 数据库的租户名。
+database-name | 是 | 无 | String | 待监控 OceanBase 数据库的数据库名。
+table-name | 是 | 无 | String | 待监控 OceanBase 数据库的表名。
+hostname | 否 | 无 | String | OceanBase 数据库或 OceanBbase 代理 ODP 的 IP 地址或主机名。
+port | 否 | 无 | String | Integer | OceanBase 数据库服务器的整数端口号。可以是 OceanBase 服务器的 SQL 端口号，默认值为 `2881`。或 ODP 的端口号默认值为 `2883`。
+connect.timeout |  否 | 30s | Duration | 连接器在尝试连接到 OceanBase 数据库服务器后的超时时间。
+server-time-zone |  否 | UTC | String | 数据库服务器中的会话时区，例如 `"Asia/Shanghai"`。此选项控制 OceanBase 数据库中的 `TIMESTAMP` 类型在快照读取时如何转换为`STRING`。确保此选项与 `oblogproxy` 的时区设置相同。
+rootserver-list | 是 | 无 | String | 格式为 `ip:rpc_port:sql_port`。多个服务器地址使用英文分号 `;` 隔开。
+logproxy.host | 是 | 无 | String | oblogproxy 的 IP 地址或主机名。
+logproxy.port | 是 | 无 | Integer | oblogproxy 的端口号。
```

Comment:
This table is different from the English version.

Reply:
Your source?
I'm using https://github.com/ververica/flink-cdc-connectors/blob/release-2.2/docs/content/connectors/oceanbase-cdc.md